src/Login/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { loginUser } from '../actions/authentication';
+import './style.scss';
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      errors: {}
+        }
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
in one component u use bind in constructor in another in render
pls use one convention

Reply:
ok